rfc/website-revamp.md

```diff
@@ -0,0 +1,143 @@
+# Website Revamp
+
+| | |
+|-|-|
+| **Status**     | Proposed|
+| **RFC #**      | [#110](https://github.com/spinnaker/governance/pull/110) |
+| **Author(s)**  | Aimee Ukasick ([@aimeeu](https://github.com/aimeeu)), Brian Le ([@brian-armory](https://github.com/brian-armory)), Rosalind Benoit ([@dnilasor](https://github.com/dnilasor)) |
+| **SIG / WG**   | Documentation SIG |
+
+## Overview
+
+Spinnaker.io is the public face of the Spinnaker project. This RFC proposes several improvements to .io and a pathway to achieving them.
+
+## Goals and Non-Goals
+
+### Goals
+
+The ultimate goal is a more modern looking and consistently styled spinnaker.io.
+
+To that end, these are the specific goals we are working toward:
+
+* Provide clear entryways and paths for the different personas spinnaker.io should speak to. For example, a more clear and opinionated navigation for someone brand new to Spinnaker.
+* Provide a clear and compelling answer to anyone asking, “Why Spinnaker for Continuous Delivery?”
+* Add useful documentation features, e.g. versioned documentation and a better in-page table of contents
```

Comment:
This is maybe way too specific a request for the RFC, but I'd love to see a search bar on the landing page for spinnaker.io, as  I always need to click on some random link to get the search bar before I can search for something.

Reply:
Agreed! Will add. I think we took this for granted and assumed it'd be there.